src/System.Net.WebSockets.Client/tests/ClientWebSocketTestBase.cs

```diff
@@ -17,8 +17,8 @@ namespace System.Net.WebSockets.Client.Tests
     /// </summary>
     public class ClientWebSocketTestBase
     {
-        public readonly static object[][] EchoServers = Configuration.WebSockets.EchoServers;
-        public readonly static object[][] EchoHeadersServers = Configuration.WebSockets.EchoHeadersServers;
+        public readonly static object[][] EchoServers = System.Net.Test.Common.Configuration.WebSockets.EchoServers;
```

Comment:
Is this because of some references from the TargetingPack folder getting auto-added? I wonder if we can avoid changing the test code if we exclude some references.